src/sentry/tasks/post_process.py

```diff
@@ -172,7 +172,7 @@ def index_event_tags(project_id, event_id, tags, group_id=None, **kwargs):
         )
 
         tagvalue, _ = TagValue.objects.get_or_create(
-            project=Project(id=project_id),
+            project=Project.objects.get(id=project_id),
```

Comment:
you'd have to take this out of using get_or_create and do the transaction manually

```
try:
    with transaction.atomic():
        do_create
except IntegrityError:
    do_get
```

Then in the create call I believe you can use project_id=ID, and in the get call you can do project=ID.

(why did the Project(id=X) not work btw?)